### PR TITLE
test(sink): update the link to download spark tgz for iceberg test

### PR DESCRIPTION
**Pull request by wenym1**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

Fix https://github.com/risingwavelabs/risingwave/issues/8093

## Checklist For Contributors

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features).
- [x] I have demonstrated that backward compatibility is not broken by breaking changes and created issues to track deprecated features to be removed in the future. (Please refer to the issue)
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Checklist For Reviewers

- [x] I have requested macro/micro-benchmarks as this PR can affect performance substantially, and the results are shown.

## Documentation

- [x] My PR **DOES NOT** contain user-facing changes.

<details><summary>Click here for Documentation</summary>

### Types of user-facing changes

Please keep the types that apply to your changes, and remove the others.

- Installation and deployment
- Connector (sources & sinks)
- SQL commands, functions, and operators
- RisingWave cluster configuration changes
- Other (please specify in the release note below)

### Release note

</details>

### CODE REVIEW

File: ci/build-ci-image.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ export RUST_TOOLCHAIN=$(cat ../rust-toolchain)
 # !!! CHANGE THIS WHEN YOU WANT TO BUMP CI IMAGE !!! #
 #          AND ALSO docker-compose.yml               #
 ######################################################
-export BUILD_ENV_VERSION=v20230220
+export BUILD_ENV_VERSION=v20230221_02
```
Comment (wcy-fdu) [inline ci/build-ci-image.sh:17]:
```suggestion
export BUILD_ENV_VERSION=v20230221_iceberg
```

**Review by xxchan — COMMENTED** [review]
Would it hurt maintainability to use separated docker image? (& spark is 200m so we don't want to put it in base image) I feel s3 artifact might be better.

**Review by xxchan — APPROVED** [review]

**Comment by xxchan** [comment]
```
spark-3.3.1-bin-had 100%[===================>] 285.48M  61.3MB/s    in 4.6s
```

Not extremely fast bust acceptable 🤔 (Last time I tested it's ~200MB/s #7870 (Although I expect it to be xGB/s...

**Review by Li0k — APPROVED** [review]

**Review by jon-chuang — APPROVED** [review]
LGTM. Hope it does not incur too much AWS cost (egress) for us.

**Pull request merged**